python/docs/source/getting_started/install.rst

```diff
@@ -55,27 +55,27 @@ For PySpark with/without a specific Hadoop version, you can install it by using
 
 .. code-block:: bash
 
-    PYSPARK_HADOOP_VERSION=2.7 pip install pyspark
+    PYSPARK_HADOOP_VERSION=2 pip install pyspark
 
-The default distribution uses Hadoop 3.2 and Hive 2.3. If users specify different versions of Hadoop, the pip installation automatically
+The default distribution uses Hadoop 3 and Hive 2.3. If users specify different versions of Hadoop, the pip installation automatically
```

Comment:
not sure if we should be more accurate and say `The default distribution uses Hadoop 3.3 ..`

Reply:
Yea, should be accurate